pkg/controller/import-controller.go

```diff
@@ -208,9 +210,18 @@ func (r *ImportReconciler) reconcilePvc(pvc *corev1.PersistentVolumeClaim, log l
 			// Don't create the POD if the PVC is completed already
 			log.V(1).Info("PVC is already complete")
 		} else if pvc.DeletionTimestamp == nil {
-			// Create importer pod, make sure the PVC owns it.
-			if err := r.createImporterPod(pvc); err != nil {
-				return reconcile.Result{}, err
+
+			_, ok := pvc.Annotations[AnnImportPod]
```

Comment:
This is more a stylistic thing than a code problem, but I would write this as:
```go
if _, ok := pvc.Annotations[AnnImportPod]; ok {
...
} else {
...
}
```
Second, what is the difference between initPvcPodName vs createImporterPod. I see initPvcPodName is creating the name and adding it to the annotations, but can't we do that in createImporterPod as well? I am trying to understand why this is split.

Reply:
This was started with a plan to change name generation logic, but now as we can see the name generation is backwards compatible, so this does not make sense/does not help, I can simplify